packages/cli/test/scripts/push.test.js

```diff
@@ -87,8 +87,8 @@ contract('push script', function([_, owner]) {
     it('should deploy and link contracts that require libraries', async function () {
       const address = this.networkFile.contract('WithLibraryImpl').address;
       const deployed = await WithLibraryImplV1.at(address);
-      const result = await deployed.double(10);
-      result.toNumber().should.eq(20);
+      const result = await deployed.methods.double(10).call();
+      parseInt(result, 10).should.eq(20);
```

Comment:
For the future, I'd consider having a custom helper instead of plain `parseInt`. I wouldn't rule out the fact that web3 may switch to native bignumbers instead of using strings, and then we would need a different function for parsing them. Centralizing how we go from web3-integers (currently strings) to real ints would help in these scenarios.